src/store/reducers/ethereum/page.ts

```diff
@@ -0,0 +1,13 @@
+import { ETHEREUM_FETCH_MORE_TRANSACTIONS } from "../../actions/ethereum/page";
+import { Block } from "../../../types";
+
+const initState = 1;
```

Comment:
Jak state jest tylko jedną wartością to może lepiej ją nazwać tak żeby było wiadomo co to jest? No chyba że konsekwencja z nazwą state jest ważniejsza ale może wtedy nie zaszkodziło by się trzymać tego że raczej jest obiektem i dać jedną wartość w środku. Taki detal ale jak zobaczyłem `state + 1` to tak lekko się zdziwiłem co tam może się dziać

Reply:
Moglbym to nazwac `counter`, ale jakos mi to nie pasuje i nie czuje, ze dodaje cokolwiek do zrozumienia. Jest komentarz wyjasniajacy kiedy loader sie pokazuje, a kiedy chowa i myslalem, ze to wystarczy.

Caly reducer ma doslownie 1 wartosc. Nie jestem za dodawaniem do obiektu pola `counter`, kiedy nic wiecej nie jest planowane do dodawania do niego. :D

Reply:
Spoko, może być 😄

Reply:
W sumie gdyby dodac do loadera pole `elementsLoading` i na tym operowac, to chyba odrazu wszystko by bylo jasne, co nie? Zobacze co inni powiedza.